### DISCUSSION TITLE
Slashes instead of letters

### DISCUSSION THREAD

**Discussion by arliber** [starship/starship#2823] [Q&A] [answered]
Hi everyone, I'm on a Macbook Pro M1 with zsh and iTerm2 (the issue however happens in other terminals, like VS Code).

Instead of letters I see `[\]`, like so:

![image](https://user-images.githubusercontent.com/226001/123482260-9f45c200-d60d-11eb-8ce1-96310987cda8.png)

I made sure to install a nerd font and enable it in iTerm's settings.

Any idea what am I missing?

Thank you

**Comment by matchai** [comment, marked as answer]
Hey @arliber 👋 

It seems like the prompt is formatting the output for the wrong shell in use. This can often happen because the `STARSHIP_SHELL` environment variable is being set after the initial setup, where it is set for your prompt.

I'd recommend clearing the variable or setting it to `zsh` if that is the prompt being used.

Here's how the differently formatted shell outputs appear for me in fish:
<img width="1393" alt="CleanShot 2021-06-26 at 18 12 13@2x" src="https://user-images.githubusercontent.com/4658208/123527041-17fa5c00-d6aa-11eb-81c6-68787e21632d.png">

**Reply by arliber** [reply]
Hi @matchai ! Thank you much! Not only the varaible was missing but the initialization script said `eval "$(starship init bash)"` instead of `zsh`. I must have carried this over from my previous Mac.
Thanks!

**Reply by matchai** [reply]
Good find! You shouldn't need to manually set the environment variable, as it should be set by the init script.
Glad to hear it's working now!
